Dockerfile-dev

```diff
@@ -9,6 +9,8 @@ ADD resources/secrets.edn.dummy /root/.secrets.edn
 
 ENV ENVIRONMENT development
 ENV PROFILE dev-docker
+ENV AWS_ACCESS_KEY_ID foo
+ENV AWS_SECRET_ACCESS_KEY bar
```

Comment:
AWS SDK complains that it can't find any credentials before it even attempts to connect to the endpoint so we give it some bogus credentials.